crates/pipeline-manager/src/api/main.rs

```diff
@@ -43,7 +43,7 @@ use utoipa_swagger_ui::SwaggerUi;
     modifiers(&SecurityAddon),
     info(
         title = "Feldera API",
-        description = r"
+        description = r#"
```

Comment:
isn't there a notion of "resource" in Rust?
In java if you have such strings you can put them in a separate file which is packaged in the generated binary (jar)
This would also enable you to edit the string separately and use the IDE to check for format problems

Reply:
moreover, it could be useful to have this file somewhere in our documentation as well

Reply:
> isn't there a notion of "resource" in Rust?

I'm sure there is a crate that does this (i.e., statically inline strings from file, or at least dynamically), but I don't think there is such a concept in the standard library. I would say it's for this PR out of scope, but potentially useful to add in the future.

It gets generated at https://docs.feldera.com/api/ , so it is in the documentation like this as well -- for now I would say that's a good enough starting place, and if we get more questions, we could link it somewhere else as well.